tgui/packages/tgui/interfaces/CryopodConsole.js

```diff
@@ -0,0 +1,91 @@
+import { useBackend } from '../backend';
+import { Button, Section, NoticeBox, LabeledList, Collapsible } from '../components';
+import { Window } from '../layouts';
+
+export const CryopodConsole = (props, context) => {
+  const { data } = useBackend(context);
+  const { account_name, allow_items } = data;
+
+  return (
+    <Window title="Cryopod Console">
+      <Window.Content>
+        <Section title={`Hello, ${account_name || '[REDACTED]'}!`}>
+          This automated cryogenic freezing unit will safely store your
+          corporeal form until your next assignment.
+        </Section>
+        <CrewList />
+        {!!allow_items && <ItemList />}
+      </Window.Content>
+    </Window>
+  );
+};
+
+const CrewList = (props, context) => {
+  const { data } = useBackend(context);
+  const { frozen_crew } = data;
+
+  return (
+    <Collapsible title="Stored Crew">
+      {!frozen_crew.length ? (
+        <NoticeBox>No stored crew!</NoticeBox>
+      ) : (
+        <Section height={10} fill scrollable>
+          <LabeledList>
+            {frozen_crew.map((person) => (
+              <LabeledList.Item key={person} label={person.name}>
+                {person.rank}
+              </LabeledList.Item>
+            ))}
+          </LabeledList>
+        </Section>
+      )}
+    </Collapsible>
+  );
+};
+
+const ItemList = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { frozen_items } = data;
+
+  const replaceItemName = (item) => {
+    let itemName = item.toString();
+    if (itemName.startsWith('the')) {
```

Comment:
will this catch words that start with `the` such as `thematic` or something? You may want to also check for the whitespace immediately after `the`

Reply:
Yes it will, space added